src/bin/rover.rs

```diff
@@ -0,0 +1,13 @@
+use rover::*;
+
+use anyhow::Result;
+use structopt::StructOpt;
+
+fn main() -> Result<()> {
+    logger::init();
+
+    let cli = cli::Apollo::from_args();
+    log::debug!("Command structure {:?}", cli);
+    cli.run()?;
+    Ok(())
```

Comment:
What about the number exit codes? Did we not carry that over or will that come later?

Reply:
come later